test/Microsoft.TemplateEngine.Cli.UnitTests/TemplateResolutionTests/TemplateMatchInfoTests.cs

```diff
@@ -11,7 +12,7 @@ public class TemplateMatchInfoTests
         [Fact(DisplayName = nameof(EmptyMatchDisposition_ReportsCorrectly))]
         public void EmptyMatchDisposition_ReportsCorrectly()
         {
-            ITemplateInfo templateInfo = new TemplateInfo();
+            ITemplateInfo templateInfo = new MockTemplateInfo();
```

Comment:
minor:
design question, in these cases, instantiating via new vs. factory.  If you wanted to later used distinct mocks, you'd not have to refactor.

Reply:
So far there is a single class for template info, however point taken., thank you.